sources/System/Health/HealthSystem.cpp

```diff
@@ -0,0 +1,26 @@
+//
+// Created by Tom Augier on 2021-05-20.
+// Edited by Benjamin Henry on 2021-05-20.
+//
+
+#include "HealthSystem.hpp"
+#include "Component/Health/HealthComponent.hpp"
+#include "Component/Controllable/ControllableComponent.hpp"
+#include "Entity/Entity.hpp"
+
+namespace BBM
+{
+	HealthSystem::HealthSystem()
+		: WAL::System({
+			typeid(HealthComponent)
+		})
+	{}
+
+	void HealthSystem::onFixedUpdate(WAL::Entity &entity)
+	{
+		auto &health = entity.getComponent<HealthComponent>();
+
+		if (health.getHealthPoint() == 0);
+			entity.setDisable(true);
```

Comment:
why ?

Reply:
Good question, it was on the branch I based myself on. I will change this.